lib/sso_users_api/manager_job.rb

```diff
@@ -1,30 +1,16 @@
-require 'active_job'
+require 'sidekiq'
 
 module SsoUsersApi
-  class ManagerJob  < ActiveJob::Base
-    queue_as :sso_users_api
+  class ManagerJob
+    include Sidekiq::Worker
 
-    def self.delay_amount
-      2
-    end
+    sidekiq_options queue: :high_priority, retry: 0
```

Comment:
what does the retry: 0 do? Can you add a comment?

Reply:
Actually, that should be the same as `retry: false`, i.e. it disables the job from retrying. But I think this should be removed and that it's safe to retry this job.